source/includes/oai-pmh/_root.md

```diff
@@ -143,28 +187,66 @@ correct community identifier.
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite&set=user-cfa)
 
-If you need selective harvesting and your use case is not supported by above
-sets, please [contact us](http://about.zenodo.org/contact/) and we may possible
-create a specific set for you.
 
-## Update schedule
+```python
+# Fetch a couple of records from the OAI Set of the "cfa" community
+>>> records = sickle.ListRecords(metadataPrefix='oai_dc', set='user-cfa')
+>>> record = records.next()
 
-Most updates are available immediately, some few updates are only reflected in the OAI sets once an hour.
+# To inspect on what sets a record is in
+>>> record.header.setSpecs
+['openaire_data', 'user-cfa']
+```
 
-## Changes
+### Harvesting with a different metadata format
+There is also the possibility of using different metadata formats. For that, we only need to replace the `metadataPrefix` argument.
 
-**2016-09-12**
+[See example](https://zenodo.org/oai2d?verb=ListRecords&
+metadataPrefix=oai_datacite&set=user-cfa)
+
+```python
+# Community harvest using "oai_datacite" metadata format
+>>> records = sickle.ListRecords(metadataPrefix='oai_datacite', set='user-cfa')
+>>> record = records.next()
+
+# Retrieving metadata from the record
+>>> record.metadata['title']
+['Computing and Using Metrics in ADS']
 
-- Deprecated metadata formats `datacite3` and `oai_datacite3`. End of life is February 2018.
-- Removed rate limit of 0.5 requests/second.
-- Changed resumption token expiry to two minutes due to major update of our
-  underlying repository software.
+>>> record.metadata['creatorName']
+['Henneken, Edwin']
 
-**2014-03-10**
+>>> record.metadata['identifier']
+['10.5281/zenodo.10897']
+```
+
+
+### Harvesting with multiple filters
+Using multiple filters to harvest records enables a higher level of granularity, allowing us to retrieve specific groups of records.
```

Comment:
```suggestion

Using multiple filters to harvest records enables a higher level of granularity, allowing us to retrieve specific groups of records.
```

Newline after titles :) You can also add something like [a Markdown linter](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in your editor which helps to catch these.

Reply:
Thank you for the suggestion!